managed/src/main/java/com/yugabyte/yw/controllers/TablesController.java

```diff
@@ -97,17 +98,8 @@ public Result create(UUID customerUUID, UUID universeUUID) {
       resultNode.put("taskUUID", taskUUID.toString());
       auditService().createAuditEntry(ctx(), request(), Json.toJson(formData.data()));
       return Results.status(OK, resultNode);
-    } catch (NullPointerException e) {
```

Comment:
You need to see where NPE is coming from.

Reply:
taskParams.tableDetails is coming as null :) -> line no 71
Will add null check for this one and throw Bad_request.